pom.xml

```diff
@@ -2717,7 +2717,6 @@
                   <rules>
                     <bannedDependencies>
                       <excludes combine.children="append">
-                        <exclude>*:*_2.11</exclude>
                         <exclude>*:*_2.10</exclude>
```

Comment:
Not quite -- the rule has to stay on the 2.12 profile, because it's the one that needs to exclude _2.11 dependencies. The exclusion for _2.10 is already in the parent rule and can be removed. But that's the only change here.
Yes the version changes look right.

Reply:
Make sense. I made the parent rule to exclude 2.10, and moved the exclusion of 2.11 to 2.12 profile. Thanks.

Reply:
Yeah that looks right to me.